content/04.workflows-and-software.md

```diff
@@ -76,26 +75,30 @@ To meet this need, most workflow managers integrate with software management sys
 Software management systems perform some combination of software installation, management, and packaging that alleviate problems that arise from dependencies and that facilitate documentation of software versions.
 On many systems, system-wide software management is overseen by system administrators, who ensure commonly-used and requested software is installed into a "module" system available to all users.
 Unfortunately, this system does not lend itself well for exploring new workflows and software, as researchers do not have permission to install software themselves.
-The Conda package manager has emerged as a leading software installation and management solution, largely because it handles both cluster permission and version conflict issues with a user-based software environment system, and features a straightforward "recipe" system which simplifies the process of making new software installable (**Figure @fig:conda_figure**).
-Conda enables lightweight software installation and can be used with the same commands across platforms.
-Alternatively, container solutions like docker and singularity allow for for the entire computational environment to be captured and distributed, including the operating system.
-This ensures that an environment is completely reproducible, and is common for production workflows.
+The Conda package manager has emerged as a leading solution, largely because it handles both cluster permission and version conflict issues with a user-based software environment system, and features a straightforward "recipe" system which simplifies the process of making new software installable (**Figure @fig:conda_figure**).
+Conda enables lightweight software installation and can be used with the same commands across platforms, but can still be impacted by differences in the host operating system.
+Encasing software environments within "containers" that capture and reproduce all other aspects of the runtime environment can enhance reproducibility over time.
```

Comment:
beautiful transition!

Reply:
👍!
Forgot to add Gruning 2018 reference here though- could you add after "host operating system"? idk how to edit your edits. doi 10.1016/j.cels.2018.03.014